apps/src/music/views/MusicView.jsx

```diff
@@ -484,7 +484,12 @@ class UnconnectedMusicView extends React.Component {
     const imageAttributions = this.library.getImageAttributions();
     if (imageAttributions.length > 0) {
       setExtraCopyrightContent(
-        <ImageAttributions attributions={this.library.getImageAttributions()} />
+        <>
+          <ImageAttributions
+            attributions={this.library.getImageAttributions()}
+          />
+          <p>Audio editing and original sounds by William Cremin.</p>
```

Comment:
@wilkie Will the new localization system take care of this, or should we localize through the older system?